### PR TITLE
sysPrxForUser: improve crash dump functions

### PR DESCRIPTION
**Pull request by Megamouse**
this might fix some crashes that could appear in the todo logging itself

### CODE REVIEW

File: rpcs3/Emu/Cell/Modules/sysPrxForUser.h [hunk 1]
```diff
@@ -42,6 +42,19 @@ struct sys_lwmutex_locker
...
+{
+	SYS_CRASH_DUMP_MAX_LABEL_SIZE = 16,
+	SYS_CRASH_DUMP_MAX_LOG_AREA   = 127 // not actually defined in CELL
+};
+
+struct sys_crash_dump_log_area_info_t
+{
+	char label[SYS_CRASH_DUMP_MAX_LABEL_SIZE]; // 15 + 1 (0 terminated)
+	vm::ptr<void> addr{ vm::null };
+	be_t<u64> size{ 0 };
```
Comment (elad335) [inline rpcs3/Emu/Cell/Modules/sysPrxForUser.h]: Why are there default initializers here? its not like we are going to construct objects from this struct.

Comment (Nekotekina) [reply rpcs3/Emu/Cell/Modules/sysPrxForUser.h]: True, you can always zero it by empty aggregate initialization.

Comment (Megamouse) [reply rpcs3/Emu/Cell/Modules/sysPrxForUser.h]: Is it actually a problem or just a nitpick? :t

Comment (elad335) [reply rpcs3/Emu/Cell/Modules/sysPrxForUser.h]: A nitpick, but its better to change it.

Comment (Megamouse) [reply rpcs3/Emu/Cell/Modules/sysPrxForUser.h]: Okay i'll change it later

**Pull request merged**